src/strings/chat_subtitle_string.rs

```diff
@@ -0,0 +1,205 @@
+    }
+}
+
+glib::wrapper! {
+    pub(crate) struct ChatSubtitleString(ObjectSubclass<imp::ChatSubtitleString>);
+}
+
+impl ChatSubtitleString {
+    pub(crate) fn new(chat: Chat, should_show_actions: bool) -> ChatSubtitleString {
+        let obj: ChatSubtitleString = glib::Object::builder().build();
+
+        if should_show_actions {
+            chat.actions()
+                .connect_items_changed(clone!(@weak obj => move |_, _, _, _| {
+                     obj.notify("subtitle");
+                }));
+        };
+
+        chat.connect_notify_local(
+            Some("online-member-count"),
+            clone!(@weak obj => move |_, _| {
+                obj.notify("subtitle");
+            }),
+        );
+        match chat.type_() {
+            ChatType::BasicGroup(basic) => {
+                basic.connect_notify_local(
+                    Some("member-count"),
+                    clone!(@weak obj => move |_, _| {
+                        obj.notify("subtitle");
+                    }),
+                );
+            }
+            ChatType::Supergroup(group) => {
+                group.connect_notify_local(
+                    Some("member-count"),
+                    clone!(@weak obj => move |_, _| {
+                        obj.notify("subtitle");
+                    }),
+                );
+            }
+            ChatType::Private(user) if !chat.is_own_chat() => {
+                let user_status_string = UserStatusString::new(user.clone());
+                user_status_string.connect_notify_local(
+                    Some("string"),
+                    clone!(@weak obj => move |_, _| {
+                        obj.notify("subtitle");
+                    }),
+                );
+                obj.imp()
+                    .user_status_string
+                    .set(user_status_string)
+                    .unwrap();
+            }
+            ChatType::Secret(secret) => {
+                let user = secret.user();
+                let user_status_string = UserStatusString::new(user.clone());
+                user_status_string.connect_notify_local(
+                    Some("string"),
+                    clone!(@weak obj => move |_, _| {
+                        obj.notify("subtitle");
+                    }),
+                );
+                obj.imp()
+                    .user_status_string
+                    .set(user_status_string)
+                    .unwrap();
+            }
+            _ => (),
+        }
+
+        obj.imp().show_actions.set(should_show_actions);
+        obj.imp().chat.set(chat).unwrap();
+        obj
+    }
+
+    pub(crate) fn subtitle(&self) -> String {
+        let imp = self.imp();
+        let chat = imp.chat.get().unwrap();
+        let should_show_actions = imp.show_actions.get();
+
+        if let Some(action) = chat.actions().last() {
+            should_show_actions
+                .then(|| strings::chat_action(&action))
+                .unwrap_or(String::new())
```

Comment:
`unwrap_or_default()`

Reply:
right, it's the same, but looks much better